stdlib/public/core/ArrayBuffer.swift

```diff
@@ -575,7 +575,7 @@ extension _ArrayBuffer {
   
   @inlinable @_alwaysEmitIntoClient
   internal func getAssociatedBuffer() -> _ContiguousArrayBuffer<Element>? {
-    let getter = unsafeBitCast(
+    let getter = unsafe unsafeBitCast(
```

Comment:
We're going to need condfail mitigations in all the fragile functions, right?

Reply:
Yeah, I had an idea on that... since we have a swift-syntax tree for the inlined text, I think we can write a visitor that replaces `UnsafeExprSyntax` nodes with their argument.

Reply:
I'm glad my SwiftIfConfig rewrite can be useful :)